src/main.go

```diff
@@ -22,7 +22,7 @@ func NewTemporalCloudProvider(temporalCloudAdapter ITemporalCloudAdapter, config
 
 	err := temporalCloudAdapter.Flags().Parse(os.Args)
 	if err != nil {
-		return nil, fmt.Errorf("unable to parse flags: %v", err)
+		return nil, fmt.Errorf("failed to parse flags: %v", err)
```

Comment:
Consider using consistent phrasing for error messages. Ensure that the change from 'unable to' to 'failed to' is applied consistently across all functions in this project for uniformity.